apps/testingaccessibility/.env.template

```diff
@@ -1,28 +1,10 @@
-STRIPE_SECRET_TOKEN=sk_test_token
-BUNDLE_BUDDY_TOKEN=bundle-buddy-token
-
-NEXT_PUBLIC_PRO_SLUG=next-public-pro-slug
-NEXT_PUBLIC_BOOK_SLUG=next-public-book-slug
-
-NEXT_PUBLIC_DISABLE_LOGIN_FOR_DEV=next-public-disable-login-for-dev
-
-NEXT_PUBLIC_CLIENT_ID=next-public-client-id
-NEXT_PUBLIC_SITE_NAME=next-public-site-name
-NEXT_PUBLIC_SITE_NAME_SHORT=next-public-site-name-short
-NEXT_PUBLIC_SUPPORT_EMAIL=next-public-support-email
-NEXT_PUBLIC_AUTH_DOMAIN=http://app.egghead.af:5000
-NEXT_PUBLIC_REDIRECT_URI=http://localhost:3000/redirect
-NEXT_PUBLIC_DISABLE_LOGIN_FOR_DEV=true
-
-NEXT_PUBLIC_STRIPE_CHECKOUT_SESSIONS_CANCEL_URL=http://localhost:3000
-NEXT_PUBLIC_STRIPE_CHECKOUT_SESSIONS_SUCCESS_URL=http://localhost:3000/thanks?session_id={CHECKOUT_SESSION_ID}
-NEXT_PUBLIC_STRIPE_CHECKOUT_SESSIONS_URL=http://app.egghead.af:5000/api/v1/stripe/session?client_id=this-sites-client-id
-NEXT_PUBLIC_STRIPE_TOKEN=pk_test_token
-
-NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM=next_public_convertkit_signup_form
-NEXT_PUBLIC_CONVERTKIT_TOKEN=next_public_convertkit_token
-NEXT_PUBLIC_CONVERTKIT_SUBSCRIBER_KEY=ck_subscriber_id
-CONVERTKIT_BASE_URL=https://api.convertkit.com/v3/
```

Comment:
felt like the "template" should be focused on what you need to be up and running locally since the prod/dev env files have all this stuff anyway